bindings/java/src/org/sleuthkit/datamodel/TimelineEventType.java

```diff
@@ -231,10 +231,10 @@ public int compare(TimelineEventType o1, TimelineEventType o2) {
 					METADATA_LAST_PRINTED, METADATA_LAST_SAVED, METADATA_CREATED, PROGRAM_EXECUTION,
 					RECENT_DOCUMENTS, REGISTRY, BACKUP_EVENT_START, BACKUP_EVENT, BACKUP_EVENT_END,
 					BLUETOOTH_PAIRING, CALENDAR_ENTRY_START, CALENDAR_ENTRY_END,
-					DELETE_PROGRAM, DELETE_PROGRAM_DELETED,
+					DELETE_PROGRAM,
```

Comment:
Could we make the names of these constants consistent, e.g., PROGRAM_DELETED instead of DELETE_PROGRAM?